time.c

```diff
@@ -3814,20 +3814,16 @@ time_localtime_m(int argc, VALUE *argv, VALUE time)
 
 /*
  *  call-seq:
- *     time.gmtime    -> time
- *     time.utc       -> time
+ *    utc -> self
```

Comment:
It might be helpful to add `Time#getutc` in the `Related` section if the user wants a new Time object (and not modify the current one).

Reply:
Done.